index.html

```diff
@@ -4,18 +4,158 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <header>
+        <div class="name-title">
+          <div class="blueSquer-p">
+            <div class="blueSquer"></div> 
+            <h3 lass="firstLastName">Maya Nelson</h3>
+          </div>  
+          <p class="title">PROJECT MANAGER</p>     
+                      
+                
+        </div>
+        
+        
+        <nav role="navigation">
+            <div id="menuToggle">
+              <!--
+              A fake / hidden checkbox is used as click reciever,
+              so you can use the :checked selector on it.
+              -->
+              <input type="checkbox" />
+              
+              <!--
+              Some spans to act as a hamburger.
+              
+              They are acting like a real hamburger,
+              not that McDonalds stuff.
+              -->
+             <div class="span-1"> <span></span></div>
+             <div class="span-2"> <span></span></div>
+             <div class="span-3"> <span></span></div>
```

Comment:
Do you need to put `span` tags inside the div? 

```suggestion
             <span class="span-1"></span>
             <span class="span-2"></span>
             <span class="span-3"></span>
```

Reply:
updated :+1: